README.md

```diff
@@ -11,7 +11,7 @@ Prereq - docker & docker compose. https://docs.docker.com/install/
 $ docker-compose up -d --build
 
 $ docker exec -it phishtray_web_1 bash
-root@1ccee26bd896:/usr/src/app# python migrate
+root@1ccee26bd896:/usr/src/app# python manage.py migrate
 root@1ccee26bd896:/usr/src/app# python manage.py runserver 0.0.0.0:8000
 
```

Comment:
These will need to call to `python3.6` instead of `python`

Reply:
Also I don't think we'll need  those steps anymore as migration happens during container spin up.
We could add these commands as another section now however to the readme - quite useful.